src/VisualStudio.Roslyn.SDK/Roslyn.SDK/ProjectTemplates/CSharp/Diagnostic/CodeFix/CodeFixProvider.cs

```diff
@@ -1,34 +1,40 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Collections.Immutable;
-using System.Composition;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CodeFixes;
-using Microsoft.CodeAnalysis.CodeActions;
-using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Microsoft.CodeAnalysis.Rename;
-using Microsoft.CodeAnalysis.Text;
-
-namespace $saferootprojectname$
+﻿namespace $saferootprojectname$
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Composition;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CodeFixes;
+    using Microsoft.CodeAnalysis.CodeActions;
+    using Microsoft.CodeAnalysis.CSharp;
+    using Microsoft.CodeAnalysis.CSharp.Syntax;
+    using Microsoft.CodeAnalysis.Rename;
+    using Microsoft.CodeAnalysis.Text;
+
+    /// <summary>
+    /// Represents a basic code fix provider.
+    /// </summary>
     [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof($saferootidentifiername$CodeFixProvider)), Shared]
     public class $saferootidentifiername$CodeFixProvider : CodeFixProvider
     {
+        /// <inheritdoc />
```

Comment:
📝 This is automatic, so the comment isn't necessary

Reply:
Automatic in what sense? `StyleCop.Analyzers` will flag it as needing a comment methinks.

Reply:
Automatic in the sense that Quick Info will show the same thing either way.

Reply:
@sharwell How does one actually test the Roslyn SDK. Since it is not signed installer won't let me install the test build... ?

Reply:
You can run the integration tests via Test Explorer. One of the integration tests is named something like **VS2019 (Exp)**; running this test will just launch the IDE with your changes applied.